samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/SpeechSynthesisSamples.java

```diff
@@ -695,6 +701,87 @@ else if (result.getReason() == ResultReason.Canceled) {
         synthesizer.close();
     }
 
+    // Synthesize and save audio and subtitle (in SubRip file format, SRT).
+    public static void synthesisWordBoundaryEventToSrtAsync() throws InterruptedException, ExecutionException, IOException {
```

Comment:
I don't see a change to main.java to add this new sample to the main menu

Reply:
Yes... the main page use key A-Z to run samples and ran out of options... Maybe we need to refactor the menu to a two-level one, like what we did in python.